source/faucet/main.d

```diff
@@ -106,6 +106,7 @@ private auto splitTx (UR) (UR utxo_rng, uint count)
     static assert (isInputRange!UR);
 
     return utxo_rng
+        .filter!(tup => tup.value.output.value >= Amount(count))
```

Comment:
```suggestion
        .filter!(tup => tup.value.output.value >= Amount(count))
```
count?

Reply:
Oops.

Reply:
> count?

When `splitTx` is called, it takes the number from `Config.count`. It is the number of keys that the `Amount` is split among. However, if we have an `Amount` that is less than `count`, we get an `Output` with `Amount(0)` This triggers `assert(0)`.

https://github.com/bpfkorea/agora/blob/3402716cf39f69b31995ba205730bd189aa761ab/source/agora/utils/Test.d#L583